From 731684061ea6c90cd047941e4bc957a6b9c119b4 Mon Sep 17 00:00:00 2001
From: Andreas Stieger <Andreas.Stieger@gmx.de>
Date: Tue, 23 Apr 2024 17:51:24 +0200
Subject: [PATCH] MAINT: The OCB design is in the public domain

The inventor dedicated the design to the public domain, all patents
lapsed intentionally. Comments only, no code change.

Fixes #6403
---
 src/crypto/CryptStateOCB2.cpp | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/src/crypto/CryptStateOCB2.cpp b/src/crypto/CryptStateOCB2.cpp
index e5866b01c..16259d9df 100644
--- a/src/crypto/CryptStateOCB2.cpp
+++ b/src/crypto/CryptStateOCB2.cpp
@@ -5,11 +5,9 @@
 
 /*
  * This code implements OCB-AES128.
- * In the US, OCB is covered by patents. The inventor has given a license
- * to all programs distributed under the GPL.
- * Mumble is BSD (revised) licensed, meaning you can use the code in a
- * closed-source program. If you do, you'll have to either replace
- * OCB with something else or get yourself a license.
+ * The algorithm design was dedicated to the public domain.
+ * https://www.cs.ucdavis.edu/~rogaway/ocb/license.htm
+ * https://www.cs.ucdavis.edu/~rogaway/ocb/ocb-faq.htm
  */
 
 #include <QtCore/QtGlobal>
-- 
2.44.0

